Resources/Prototypes/Floof/Entities/Mobs/Cyborgs/quadborg.yml

```diff
@@ -17,6 +17,13 @@
       shader: unshaded
       map: ["light"]
       visible: false
+  - type: Hands
+  - type: Stripping
+  - type: Body
+    prototype: Primate
+    requiredLegs: 1 # TODO: More than 1 leg
+  - type: Inventory
+    templateId: Securityborg
```

Comment:
```suggestion
```

Reply:
This is uneeded.

Reply:
its required to give them the pocket to put handcuffs in

Reply:
Then make a custom item/tool for borgs to cuff someone.

Reply:
this has already been discussed with floof, cuffs in a module don't work, I already looked into an alternative for this.